src/libraries/System.Private.CoreLib/src/System/Number.NumberToFloatingPointBits.cs

```diff
@@ -14,23 +17,40 @@ internal unsafe partial class Number
                 exponentBits: 11,
                 maxBinaryExponent: 1023,
                 exponentBias: 1023,
-                infinityBits: 0x7FF00000_00000000
+                infinityBits: 0x7FF00000_00000000,
+                minDecimalExponent: -342,
+                maxDecimalExponent: 308,
+                infinitePower: 0x7FF,
+                minExponentRoundToEven: -4,
+                maxExponentRoundToEven: 23,
+                maxExponentFastPath: 22
             );
 
             public static readonly FloatingPointInfo Single = new FloatingPointInfo(
                 denormalMantissaBits: 23,
                 exponentBits: 8,
                 maxBinaryExponent: 127,
                 exponentBias: 127,
-                infinityBits: 0x7F800000
+                infinityBits: 0x7F800000,
+                minDecimalExponent: -65,
+                maxDecimalExponent: 38,
+                infinitePower: 0xFF,
+                minExponentRoundToEven: -17,
+                maxExponentRoundToEven: 10,
+                maxExponentFastPath: 10
             );
-
             public static readonly FloatingPointInfo Half = new FloatingPointInfo(
                 denormalMantissaBits: 10,
                 exponentBits: 5,
                 maxBinaryExponent: 15,
                 exponentBias: 15,
-                infinityBits: 0x7C00
+                infinityBits: 0x7C00,
+                minDecimalExponent: -8,
+                maxDecimalExponent: 4,
+                infinitePower: 31,
```

Comment:
nit: Use hex here to make it consistent with other `infinitePower` declarations.
```suggestion
                infinitePower: 0x1F,
```

I'm also notably not a "fan" of `infinitePower` as the name. This is really the `(InfinityBits & ExponentMask) >> ExponentShift` representing the `InfinityExponent`
* names are going off the terminology we use in `Half`/`Single`/`Double` for the internal constants, such as `SignMask`, `SignShift`, `ExponentMask`, `ExponentShift`, `ShiftedExponentMask`, etc